app/assets/javascripts/projects/canvas.js.erb

```diff
@@ -1159,7 +1159,7 @@ function updateModuleHtml(module, id, name, gridDistX, gridDistY) {
   .attr("id", id)
   .attr("data-module-id", id)
   .attr("data-module-name", name)
-  .css("z-index", "");
+  .css("z-index", "20");
```

Comment:
module.css("z-index", "20") works also with integers => module.css("z-index", 20)
please use single quote if it is possible